modules/home/assets/js/components/top-section.js

```diff
@@ -1,12 +1,40 @@
-const TopSection = () => {
+import { Box, Stack } from '@elementor/ui';
+import Typography from '@elementor/ui/Typography';
+import Button from '@elementor/ui/Button';
+import YoutubeIcon from '../icons/youtube-icon';
+
+const TopSection = ( { ctaUrl, videoUrl, embedUrl } ) => {
 	return (
-		<p>Add the TopSection component here</p>
+		<Stack direction={ { xs: 'column', md: 'row' } } sx={ { backgroundColor: 'background.default', p: { xs: 2, md: 4 } } }>
+			<Box sx={ { display: 'flex', flexDirection: 'column', pr: { xs: 0, md: 10 }, pb: 3, gap: 3 } }>
+				<Box>
+					<Typography variant="body2">{ __( 'Hi!', 'elementor' ) }</Typography>
+					<Typography variant="h6">{ __( 'Welcome to Elementor', 'elementor' ) }</Typography>
+					<Typography variant="body2" >{ __( 'Get introduced to Elementor by watching our &ldquo;Getting Started&rdquo; video series. It will guide you through the steps needed to create your website. Then click to create your first page.', 'elementor' ) }</Typography>
```

Comment:
@hein-obox Please confirm if these texts need to be translatable? It might not be viable due to the data coming from a dynamic source.

Reply:
@louiswol94 They don't need to. I am thinking, can we delete this in the next task, to make this component dynamic?

Reply:
Conclusion: Translations for this type of pages is not yet supported. Please remove these translation functions. Or we can do it in the next task when we actually bring in the dynamic data.